CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Changes
 0.11.5 (unreleased)
 ===================
 * Added a Makefile recipe to the repository to evaluate notebooks while ignoring the output cells.
+* Cleaned up documentation to facilitate easier navigation.
+* Fixed a few notebooks that were failing Jenkins build tests.
```

Comment:
What Finch notebook is failing the Jenkins build?  Nothing is failing on Jenkins since March 17 with regards to Finch notebooks.

Reply:
Opps, does this changelog includes other PR between now and the previous release? If yes then keep the note about notebook update.